flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/TaskManagersHandler.java

```diff
@@ -53,6 +59,11 @@ public TaskManagersHandler(FiniteDuration timeout, MetricFetcher fetcher) {
 	}
 
 	@Override
+	public String[] getPaths() {
+		return new String[]{TASKMANAGERS_REST_PATH, TASKMANAGER_DETAILS_REST_PATH};
```

Comment:
Just to double check: The handler is OK to handle both requests, right? Before we created two instances of the handler. I think this is actually a case where this works better than before.